tests/cases/php-langspec/exception_handling/odds_and_ends.php

```diff
@@ -20,14 +20,22 @@ class X {}	// not derived from Exception
 }
 //*/
 ///*
-catch (int $e) {		// Not diagnosed; bad
-	echo "L0: In catch-block int\n";
-}
+//catch (int $e) {		// Diagnosed; good
+//	echo "L0: In catch-block int\n";
+//}
 //*/
 ///*
 catch (X $e) {		// Not diagnosed; bad
 	echo "L0: In catch-block int\n";
 }
+
+//catch (InvalidArgumentException | $e) { // Diagnosed; good
+//    echo "L0: In catch-block multiple Exception\n";
+//}
+
+catch (BadMethodCallException | InvalidArgumentException $e) {
+    echo "L0: In catch-block multiple Exception\n";
+}
```

Comment:
Some of the grammar tests from this file were failing. This was intentionally added to test if a valid case would pass. Want this one removed?